<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>573</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDockWindow">
    <property name="title">
     <string>Окна</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuDockWindow"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dock_widget_exec">
   <property name="windowTitle">
    <string>Выполнение скрипта</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="button_exec">
       <property name="text">
        <string>Выполнить</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="code"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_widget_simple_log">
   <property name="windowTitle">
    <string>Простой лог</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="clear_slog">
       <property name="toolTip">
        <string>Очистить лог</string>
       </property>
       <property name="statusTip">
        <string>Очистить лог</string>
       </property>
       <property name="text">
        <string>Очистить лог</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="simple_log"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarGeneral">
   <property name="windowTitle">
    <string>General</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clear_slog</sender>
   <signal>clicked()</signal>
   <receiver>simple_log</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
